src/views/view.c

```diff
@@ -1019,9 +1019,11 @@ dt_view_surface_value_t dt_view_image_get_surface(int imgid, int width, int heig
   }
 
   // so we create a new image surface to return
-  const float scale = fminf(width / (float)buf_wd, height / (float)buf_ht) * darktable.gui->ppd_thb;
-  const int img_width = buf_wd * scale;
-  const int img_height = buf_ht * scale;
+  float scale = fminf(width / (float)buf_wd, height / (float)buf_ht) * darktable.gui->ppd_thb;
+  const int img_width = buf_wd * scale + 0.5f;
```

Comment:
isn't this equivalent to : 

const int img_width = roundf((float)buf_wd * scale);

?

Likewise below.

Reply:
yep, you're right, and roundf is more explicit ! Done

Reply:
Almost fixed for me.
This fixes the alignment in lightable culling mode, but in darktable it behaves weird:
When I open an image for the first time in darkroom, the thumbnail is displayed correctly while darktable reports: "working loading IMG_xxx"
Then, darktable displays: "working" and this misaligns the picture again by 1 pixel

Reply:
oops... I've forgot that case. PR #9117 should fix it.
Thanks for the tests !